src/Tooltip/Tooltip.js

```diff
@@ -1,71 +1,75 @@
-        background: none;
-        padding: 0;
-      }
-    }
-
-    .tippy-popper[x-placement='bottom'] .tippy-tooltip.arbor-theme [x-arrow] {
-      border-bottom-color: ${arrowColor};
-    }
-
-    .tippy-popper[x-placement='top'] .tippy-tooltip.arbor-theme [x-arrow] {
-      border-top-color: ${arrowColor};
-    }
-
-    .tippy-popper[x-placement='left'] .tippy-tooltip.arbor-theme [x-arrow] {
-      border-left-color: ${arrowColor};
-    }
-
-    .tippy-popper[x-placement='right'] .tippy-tooltip.arbor-theme [x-arrow] {
-      border-right-color: ${arrowColor};
-    }
-  `;
-};
+import Tippy from '@tippyjs/react';
+import 'tippy.js/dist/tippy.css';
+import { animateFill, sticky } from 'tippy.js';
+import styled from '@emotion/styled';
 
-const Tooltip = ({ content, children, theme, ...props }) => {
-  const tooltipContent = (
-    <ThemeProvider theme={theme}>
-      <StyledTooltipContent>{content}</StyledTooltipContent>
-    </ThemeProvider>
-  );
+const StyledTippy = styled(Tippy)`
+  background: none;
+  padding: 0;
+
+  &[data-placement^='top'] > .tippy-arrow::before {
+    border-top-color: ${(props) => props.theme.colors.monochrome.grey90};
+  }
+
+  &[data-placement^='bottom'] > .tippy-arrow::before {
+    border-bottom-color: ${(props) => props.theme.colors.monochrome.grey90};
+  }
+
+  &[data-placement^='left'] > .tippy-arrow::before {
+    border-left-color: ${(props) => props.theme.colors.monochrome.grey90};
+  }
+
+  &[data-placement^='right'] > .tippy-arrow::before {
+    border-right-color: ${(props) => props.theme.colors.monochrome.grey90};
+  }
+
+  .tippy-content {
+    background: ${(props) => props.theme.colors.monochrome.grey90};
+    border-radius: ${(props) => props.theme.space.small};
+    color: ${(props) => props.theme.colors.monochrome.white};
+    font-size: ${(props) => props.theme.fontSizes.size4};
+    padding: ${(props) => props.theme.space.smaller};
+    font-family: ${(props) => props.theme.brandFont};
+    line-height: ${(props) => props.theme.space.small};
+  }
+`;
+
+const Tooltip = ({ content, children, render, ...props }) => {
+  const [mounted, setMounted] = useState(false);
+
+  const lazyPlugin = {
+    fn: () => ({
+      onShow: () => setMounted(true),
+      onHidden: () => setMounted(false)
+    })
+  };
+
+  const renderProps = {};
+
+  if (render) {
+    renderProps.render = (...args) => (mounted ? render(...args) : '');
+  } else {
+    renderProps.content = mounted ? content : '';
+  }
 
   return (
-    <>
-      <Global styles={globalTippyStyles} />
-
-      <TippyTooltip
-        {...{
-          theme: 'arbor',
-          arrow: true,
-          html: tooltipContent,
-          distance: 8,
-          duration: 300,
-          updateDuration: 0,
-          arrowSize: 'medium',
-          animation: 'fade',
-          inertia: true,
-          animateFill: false,
-          ...props
-        }}
-      >
-        {children}
-      </TippyTooltip>
-    </>
+    <StyledTippy
```

Comment:
The tippyjs package supports styled system out of the box. So we don't need the global theme anymore.